TW-1476 Doc Postbot November changes (#5207)
postmanlabs/postman-docs: src/pages/docs/writing-scripts/test-scripts.md

1. Select the play button. Postbot writes a test for you.

> You can also use Postbot to autocomplete test code. If you have a response available and type `pm.test`, Postbot will suggest tests for your request. Select a test name, and Postbot will provide the code to validate what you need.

Reviewer: ```suggestion
You can also use Postbot to autocomplete test code. If you have a response available and type `pm.test`, Postbot suggests tests for your request. Select a test name, and Postbot inserts the code to validate your response.
```
